Documentation/ceph-osd-mgmt.md

```diff
@@ -66,6 +66,18 @@ and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly c
 Update your CephCluster CR. Depending on your CR settings, you may need to remove the device from the list or update the device filter.
 If you are using `useAllDevices: true`, no change to the CR is necessary.
 
+**IMPORTANT: On host-based clusters, you may need to stop the Rook Operator while performing OSD
+removal steps in order to prevent Rook from detecting the old OSD and trying to re-create it.**
```

Comment:
```suggestion
removal steps in order to prevent Rook from detecting the old OSD and trying to re-create it before the disk is cleaned.**
```

Reply:
changed to say "before the disk is wiped or removed"